package.json

```diff
@@ -6,6 +6,7 @@
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "start": "react-native start",
+    "clean": "npx react-native clean-project-auto && npx pod-install && watchman watch-del-all && npm start",
```

Comment:
Couple things here

* I think `clean-project-auto` already does `watchman watch-del-all` according to the [docs](https://github.com/pmadruga/react-native-clean-project#content)
* I would not expect a `clean` task to also start Metro. Maybe we should call this `clean-start`?
* `clean-project-auto` wipes all your cocoapods and the system and user pod caches, so you have to download and install them all ever time you clean. That makes this a bit extreme for my use, but no object to adding it if it's useful to you.

Reply:
Have removed `watchman watch-del-all` and renamed. I use it mainly when I am switching between branches or changing dependencies. But I do have some troubles from time to time that makes me want to clean everything.